testing/test_aux.out

```diff
@@ -40,7 +40,7 @@ RESULT skip security.firmware Could not retrieve a firmware version with nmap. C
 RESULT pass security.firmware version found: ?\xFF\xFF\x19,>u\x08\x00no
 dhcp requests 2 1
 01: ['01:brute:1', '01:macoui:1']
-02: ['02:nmap:1', '02:tls:1']
+02: ['02:hold:Timed out.', '02:nmap:1', '02:tls:1']
```

Comment:
needs to be a programatic single identified (e.g. "timeout"), not a string with spaces. Design intent is that it can be used for things like CSS or whatever, so want to keep it simple.